app/views/images/show.html.erb

```diff
@@ -1,3 +1,8 @@
 <div class="container-fluid">
+  <% if @image.tag_list.present? %>
+    <p><strong>Tags: </strong>
```

Comment:
Minor, but prefer `<strong>Tags: </strong>` on a separate line make it more structured